src/services/reverse-geocoder.php

```diff
@@ -5,5 +5,6 @@
 use SearchApi\Models as Models;
 
 interface ReverseGeocoder {
+  function get_url( Models\GeoCoordinate $coords );
```

Comment:
Is anything actually calling get_url? even inside of the `GoogleReverseGeocoder` it doesn't call it. I get the feeling that the ReverseGeocoder interface doesn't need to require a function get_url()